docs/source/data/fetching-rest.mdx

```diff
@@ -0,0 +1,421 @@
+---
+title: Fetching from REST
```

Comment:
Oh gosh. I forgot you had forked this file a little bit ago. @smyrick just added a bunch of stuff to the AS3 version of the data source docs that should probably be preserved! Added in https://github.com/apollographql/apollo-server/pull/6650 with a rename in https://github.com/apollographql/apollo-server/pull/6834